examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/dnssd/DiscoveredNodeData.java

```diff
@@ -0,0 +1,212 @@
+package com.chip.casting.dnssd;
+
+import android.net.nsd.NsdServiceInfo;
+
+import java.net.InetAddress;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class DiscoveredNodeData {
+  private static final int MAX_IP_ADDRESSES = 5;
+  private static final int MAX_ROTATING_ID_LEN = 50;
+  private static final String KEY_DEVICE_NAME = "DN";
+  private static final String KEY_DEVICE_TYPE = "DT";
+
+
+  private String hostName;
+  private String instanceName;
+  private long longDiscriminator;
+  private long vendorId;
+  private long productId;
+  private byte commissioningMode;
+  private long deviceType;
+  private String deviceName;
+  private byte rotatingId[] = new byte[MAX_ROTATING_ID_LEN];
+  private int rotatingIdLen;
+  private short pairingHint;
+  private String pairingInstruction;
+  private short port;
+  private int numIPs;
+  private List<InetAddress> ipAddresses;
+
+  public DiscoveredNodeData() { }
```

Comment:
Do we need this? When would we want to create an empty DiscoveredNodeData?

Reply:
Removed it.